src/services/Check.php

```diff
@@ -124,6 +124,26 @@ public static function displaySize(string $input): string
         }
     }
 
+    /**
+     * Check the display mode user setting
+     *
+     * @param string $input
```

Comment:
these doctags can be removed as they do not bring new information compared to type hints